commands/login.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/fatih/color"
 	"github.com/giantswarm/gsctl/client"
 	"github.com/giantswarm/gsctl/config"
-	"github.com/giantswarm/gsctl/pkce"
```

Comment:
I did a refactoring in this PR also, renaming the `pkce` package to `oidc` since the Refresh Token behaviour I introduced here is not really `pkce`. So I lifted the concept a bit higher, now `oidc` is the package that knows about things like `pkce` and `refresh token`